src/sagemaker/estimator.py

```diff
@@ -1342,8 +1352,6 @@ def create_model(
         role=None,
         image_uri=None,
         predictor_cls=None,
-        serializer=IdentitySerializer(),
-        deserializer=BytesDeserializer(),
```

Comment:
update the commit message to reflect this breaking change

Reply:
Fixed